web_hierarchy/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+#   © 2019 Kevin Kamau
+#   License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': "Web Hierarchy",
+    'version': '10.0.1.0.0',
+    'depends': [
+        'web'
+    ],
+    'author': "Odoo Community Association (OCA)",
+    'website': "https://github.com/KKamaa",
+    'category': 'Extra Tools',
```

Comment:
I do not think such a category exists, check out the `ir.module.category` records to verify.

Reply:
It exists check [web_planner](https://github.com/OCA/OCB/blob/acb74b37775870a6daab3dd77e2c15dcabd4f7ed/addons/web_planner/__manifest__.py#L6)